stable/kong/README.md

```diff
@@ -324,6 +325,16 @@ If your SMTP server requires authentication, you should the `username` and
 `smtp_password_secret` must be a Secret containing an `smtp_password` key whose
 value is your SMTP password.
 
+### Declarative Configuration
+
+The helm chart assumes to be running in declarative mode when it is started in 
+DB-less mode (`env.database: "off"`) and without the ingress controller 
+(`ingressController.enabled: false`). 
```

Comment:
I'd recommend to re-frame this as:

When deploying Kong in DB-less mode (`env.database: "off"`) and without the Ingress Controller (`ingressController.enabled: false`), Kong needs a config to run. In this case, configuration can be provided using a ConfigMap.